susemanager/bin/server-migrator.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
This script will run until the end even if there are errors at any step, which is fine as long as this is really what you want.

Reply:
Similar here.